src/main/resources/i18n/messages.properties

```diff
@@ -1642,6 +1642,11 @@ cart.error.project-not-in-cart=Project '{0}' is not in the cart.
 cart.clear=Empty Cart
 cart.noneMatchingFilter=No samples match filter
 
+ShareLayout.title=Share Samples with Another Project
```

Comment:
You can remove these 4 translations

Reply:
Sorry, thought I removed those.  Updated in 293fc39da1d6db7ef13fcffc23ce6bc6e4a00d48